### ISSUE TITLE
Şlkkk

### ISSUE THREAD

**Issue by omer4040** [omer4040/-lkkk#1]
import numpy as np
import tensorflow as tf
from tensorflow import keras
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Activation,Dense,Flatten,BatchNormalization,Conv2D,MaxPool2D,Input
from tensorflow import keras
from tensorflow.keras.preprocessing import image_dataset_from_directory
import cv2

# Opens the inbuilt camera of laptop to capture video.
cap = cv2.VideoCapture(1)
i = 0

while (cap.isOpened()):
    ret, frame = cap.read()

    # This condition prevents from infinite looping
    # incase video ends.
    if ret == False:
        break

    # Save Frame by Frame into disk using imwrite method
    cv2.imwrite('resim/adapt/Frame' + str(i) + '.jpg', frame)
    i += 1

cap.release()
cv2.waitKey(0)
cv2.destroyAllWindows()

train_data = tf.keras.preprocessing.image_dataset_from_directory(
                                                        directory='resimm',
                                                        labels='inferred',
                                                        label_mode='binary',
                                                        color_mode = 'rgb',
                                                        batch_size=64,
                                                        image_size=(256,256)
)

model = Sequential()									
model.add(Input(shape=(256,256,3)))
model.add(Conv2D(filters = 32, kernel_size = (3,3), activation='relu'))
model.add(MaxPool2D(pool_size = (2,2)))
model.add(Conv2D(filters = 64, kernel_size = (3,3), activation ='relu'))
keras.layers.MaxPooling2D(pool_size=(2, 2))
model.add(Conv2D(filters = 128, kernel_size = (3,3),activation ='relu'))
keras.layers.MaxPooling2D(pool_size=(2, 2))
model.add(Flatten())
model.add(keras.layers.Dropout(0.4))
model.add(Dense(1, activation='sigmoid'))

model.summary()										

model.compile(optimizer = "adam", loss = 'binary_crossentropy', metrics=['accuracy'])	

model.fit(train_data,batch_size=64,epochs=4,verbose=1)

model.save("sisekalem.model")

foto = keras.preprocessing.image.load_img("kalem1.jpg", target_size=(256,256))
foto_matrisi = keras.preprocessing.image.img_to_array(foto)
foto_matrisi = np.expand_dims(foto_matrisi, axis=0)
model = keras.models.load_model("sisekalem.model")
tahmin = model.predict(foto_matrisi)

if tahmin >= 0.70:
    print("Sise.")
if tahmin <= 0.30:
    print("Kalem.")
if tahmin <0.70 and tahmin >0.30:
    print("Tahmin edemiyorum.")
